pandas/tests/io/parser/test_common.py

```diff
@@ -63,13 +63,14 @@ def _set_noconvert_columns(self):
         "parse_dates": parse_dates,
         "delimiter": ",",
     }
+    parser.engine = "c"
     parser._engine = MyCParserWrapper(StringIO(data), **parser.options)
 
     result = parser.read()
     tm.assert_frame_equal(result, expected)
 
 
-def test_empty_decimal_marker(all_parsers):
+def test_empty_decimal_marker(all_parsers, pyarrow_xfail):
```

Comment:
this is very awkward, instead the xfail should be in all_parsers

Reply:
This is just used to xfail the pyarrow tests that fail. Moving it to all_parsers would xfail all pyarrow tests. Currently, I am xfailing tests that check for error messages being raised, use unsupported features, or have bad/non-standard data.